main.py

```diff
@@ -7,31 +7,32 @@
 import os
 import re
 import time
+
 from os import environ
 from os.path import dirname, join
-from typing import List, Optional, Union, Sequence
+from typing import List, Optional, Sequence, Union
 
 import asyncpg
 import discord
 import humanize
 import numpy as np
+
 from aiogithub import GitHub
 from discord.ext import commands
 from dotenv import load_dotenv
 
+import utils.library_override
+
 from utils.buttons import PersistentRespondView
 from utils.context_managers import UserLock
 from utils.decorators import event_check, in_executor, wait_ready
 from utils.ipc import StellaClient
 from utils.prefix_ai import DerivativeNeuralNetwork, PrefixNeuralNetwork
-from utils.useful import (ListCall, StellaContext, call, count_python,
-                          print_exception)
+from utils.useful import ListCall, StellaContext, call, count_python, print_exception
 
 dotenv_path = join(dirname(__file__), 'bot_settings.env')
 load_dotenv(dotenv_path)
 
-import utils.library_override
```

Comment:
this is needed lol dont remove it

Reply:
oh actually i didnt see it moved lmao ehe

Reply:
but still the position matters here, dont move it, just keep it under load_dotenv

Reply:
I'll undo this, why though? i asked you before and you didn't comment on that

Reply:
sir you asked the position of other imports, that one is fine, but this particular library_override file is important soz

Reply:
I'm asking why is it important. it's under all the 3rd party library imports that you patch and I can't see how it relies on env vars that you load from file

Reply:
cause i need to load my env before jishaku is first referenced into the memory, otherwise it wouldn't see my settings

Reply:
fixed